x-pack/plugins/apm/server/routes/agent_keys/invalidate_agent_key.ts

```diff
@@ -9,17 +9,20 @@ import { ApmPluginRequestHandlerContext } from '../typings';
 export async function invalidateAgentKey({
   context,
   id,
+  isAdmin,
 }: {
   context: ApmPluginRequestHandlerContext;
   id: string;
+  isAdmin: boolean;
 }) {
+  const body: { ids: string[]; owner?: boolean } = { ids: [id] };
+  if (!isAdmin) {
+    body.owner = true;
+  }
```

Comment:
Would this work or does the endpoint not  accept `owner: false`?

```suggestion
  const body = { ids: [id], owner: isAdmin };
```